src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/exceptions/EnvelopeNotFoundException.java

```diff
@@ -0,0 +1,4 @@
+package uk.gov.hmcts.reform.bulkscanprocessor.exceptions;
+
+public class EnvelopeNotFoundException extends RuntimeException {
+}
```

Comment:
Missing entry in `ControllerAdvice` to properly respond with 404

-- EDIT --

Ah, service is not actually plugged in anywhere yet. Can be done when plugged in controller ;)